UnitedIncome/components
# App 3239 build the new spending choice screen

### Description
Updates to the Box component to support disabled state and to show the check

Should look like: https://zpl.io/agWykrO

### Testing Instructions
1. Check out the disabled with check story for the Box component


- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3239)
